### PR TITLE
Dan cta bus merge

### PR DESCRIPTION
**Pull request by baumannalexj**

### CODE REVIEW

File: lambda/custom/index.js [hunk 1]
```diff
@@ -21,9 +22,17 @@ const handlers = {
         });
     },
     'CtaBusIntent': function () {
-        // TODO: Build proper parameters object
-
-        this.emit(':tell', "implement bus intent");
+        let transitSlot = this.event.request.intent.slots.transitMode.value;
+        if (transitSlot === "bus") {
+            let parameters = ParameterHelper.getLocationParameters(this.event.context.System);
```
Comment (rknruben56) [inline lambda/custom/index.js]: Isn't this supposed to be in the intent below on line 45?

File: lambda/custom/repositories/transit/CtaBusRepository.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            key: CTABUS_API_KEY,
+            rt: parameters.rt,
+            dir: parameters.dir,
+            format: "json"
+        }
+        
+    });
+
+    request(url,  (error, response, body) => {      
+        let stopId = closestStopId(latitude, longitude, JSON.parse(body));
```
Comment (rknruben56) [inline lambda/custom/repositories/transit/CtaBusRepository.js]: Does this return anything? I don't think the callback is being called

**Review by rknruben56 — APPROVED** [review]

File: .ask/config [hunk 3]
```diff
@@ -1,14 +1,14 @@
 {
   "deploy_settings": {
     "default": {
-      "skill_id": "amzn1.ask.skill.e0929fb0-ad82-43f5-b785-95eee4ddef38",
+      "skill_id": "amzn1.ask.skill.68bc5ea2-f393-4596-a0d8-64f9e1790a09",
```
Comment (baumannalexj) [inline .ask/config]: reminder

File: .ask/config [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
       "was_cloned": false,
       "merge": {
         "manifest": {
           "apis": {
             "custom": {
               "endpoint": {
-                "uri": "chicago-now-dev-main"
```
Comment (baumannalexj) [inline .ask/config]: reminder

File: skill.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
     "publishingInformation": {
       "locales": {
         "en-US": {
-          "name": "ChicagoNow"
+          "name": "ChicagoNow-dan"
```
Comment (baumannalexj) [inline skill.json]: reminder

**Pull request merged**